packages/nx-plugin/src/schematics/plugin/plugin.ts

```diff
@@ -116,7 +116,11 @@ function addFiles(options: NormalizedSchema): Rule {
         move(options.projectRoot)
       ]),
       MergeStrategy.Overwrite
-    )
+    ),
+    schematic('builder', {
+      project: options.name,
```

Comment:
When you add support for `unitTestRunner`, pass the option here as well.

Reply:
👍

Reply:
Were you wanting me to add that flag to the plugin schematic as well? Or just hardcode the flag as `jest` here? I have some bandwidth and can do that if need be.